Faraday_Proxy_Tools/FaradayIO/commandmodule.py

```diff
@@ -131,6 +131,55 @@ def create_fixed_length_packet(data, fixed_legth):
         return padded_data
 
 
+def create_fixed_length_packet_padding(data, fixed_legth, padding_byte):
```

Comment:
Trailing zeros is exactly what the "%.2f" above does. Notice the two different formatting statements.